src/main/java/codurance/academyfinalboy/backend/infrastructure/services/GoogleTokenValidator.java

```diff
@@ -18,6 +18,10 @@
   @Override
   public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler)
       throws Exception {
+    if (request.getMethod().equals("OPTIONS")) {
```

Comment:
@sofiacarballo-codurance I guess your question comes from this part of the code. This is because OPTIONS requests (preflight) do not send the token in the header so that is why it was failing the authentication. Remember that before sending the actual login request it sends a preflight request. If we don't exclude it from the auth middleware it will always give us a 401.